src/index.tsx

```diff
@@ -430,11 +664,7 @@ export class App extends React.Component<any, AppState> {
     } else if (event[KEYS.META] && /z/i.test(event.key)) {
       event.preventDefault();
 
-      if (
-        this.state.resizingElement ||
-        this.state.multiElement ||
-        this.state.editingElement
```

Comment:
Why removing the other 2 checks?

Reply:
Because that is the reason why undo was not working after resize. I am not sure about editingElement either; so, I removed it. I might have a solution for this but it is a bit of a big one and I want to finish some other stuff before thinking about it.